src/client/core/utils/dom.js

```diff
@@ -51,6 +51,8 @@ export const findParent                             = hammerhead.utils.dom.findP
 export const getTopSameDomainWindow                 = hammerhead.utils.dom.getTopSameDomainWindow;
 
 export const isRadioButtonElement = el => isInputElement(el) && el.type === 'radio';
+export const isColorInputElement  = el => isInputElement(el) && el.type === 'color';
```

Comment:
I don't know why it's necessary to reexport some methods from hammerhead.
Use the appropriate dom utils methods from [hammerhead](https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/utils/dom.ts#L440) at once.

Reply:
`hammerhead` does not have such methods as `isRadioButtonElement`, `isColorInputElement` or `        isCheckboxElement`

Reply:
Add them

Reply:
sounds like a separate issue in hammerhead

Reply:
Today we will publish a new hammerhead version with appropriate dom utils.